### ISSUE TITLE
What's the current status of the project?

### ISSUE THREAD

**Issue by EffiSciencesResearch** [google/unrestricted-adversarial-examples#86]
This seems to be a cool project but the last update was 2 years ago. What happened since that?

From https://ftxfuturefund.org/projects/ai-alignment-prizes/ , Solving this challenge is supposed to be worth 10M. Is that still a thing?

**Comment by isaac-dunn** [comment]
See the discussion at #85

**Comment by carlini** [comment]
Yeah I also saw that post... and I agree we should probably put some kind of update to explain what's going on here. I agree that, if this challenge would be solved, whoever won it should receive glory and fame (and money). I just am not sure that putting money behind it will make it happen in the right way any faster---or not sure enough to spend the time it would take to run the contest itself.

Note that, even if the contest isn't running as such, anyone is still free to try and attempt robust classifications of birds vs bicycles. I really do think it's important.
